openmc/executor.py

```diff
@@ -51,8 +51,44 @@ def plot_geometry(output=True, openmc_exec='openmc', cwd='.'):
         Path to working directory to run in. Defaults to the current working directory.
 
     """
+    if output:
+        output = 'full'
+    return _run([openmc_exec, '-p'], output, cwd)
 
-    return _run(openmc_exec + ' -p', output, cwd)
+
+def run_volume_calculation(threads=None, output=True, cwd='.',
+                           openmc_exec='openmc', mpi_args=None):
+    """Run stochastic volume calculations in OpenMC
```

Comment:
Perhaps this is just my naiveté shining through, but does this method just run and calculate the volume for all cells in a simulation? I presume one must have first created a "volume.xml" input file? If so, can this information be added to the docstring?